.eslintrc.js

```diff
@@ -277,5 +275,14 @@ module.exports = {
         },
       ],
     },
+
+    {
+      // Applies to only published packages
+      files: ['packages/**/*.ts'],
+      rules: {
+        // Error if comments do not adhere to `tsdoc`.
+        'tsdoc/syntax': 'error',
```

Comment:
This became too noisy, so scoped it only to `packages/` so only packages we publish need to comply for Documentation reasons.